clang/test/Driver/arm-fpu-selection.s

```diff
@@ -0,0 +1,32 @@
+// REQUIRES: arm-registered-target
+// Ensures that when targeting an ARM target with an Asm file, clang
```

Comment:
I have often been asked to use `///` for comments that were actually comments. I think it is marginally easier to read if you do that.